contrib/verifybinaries/verify.sh

```diff
@@ -149,6 +149,26 @@ for file in $FILES
 do
    echo "Downloading $file"
    wget --quiet -N "$HOST1$BASEDIR$file"
+    wget --quiet -N "$HOST1$BASEDIR$file.asc"
+
+    GPGOUT=$(gpg --verify "$file.asc" 2>&1)
+    RET="$?"
+    if [ $RET -ne 0 ]; then
+       if [ $RET -eq 1 ]; then
+          #and notify the user if it's bad
+          echo "Bad signature."
+       elif [ $RET -eq 2 ]; then
+          #or if a gpg error has occurred
+          echo "gpg error. Do you have the Dash Core binary release signing key installed?"
+       fi
+
+       echo "gpg output:"
+       # shellcheck disable=SC2001
+       echo "$GPGOUT"|sed 's/^/\t/g'
+       clean_up $SIGNATUREFILENAME $SIGNATUREFILENAME.2 $TMPFILE
+       exit "$RET"
+    fi
```

Comment:
```suggestion
   wget --quiet -N "$HOST1$BASEDIR$file.asc"

   GPGOUT=$(gpg --verify "$file.asc" 2>&1)
   RET="$?"
   if [ $RET -ne 0 ]; then
      if [ $RET -eq 1 ]; then
         #and notify the user if it's bad
         echo "Bad signature."
      elif [ $RET -eq 2 ]; then
         #or if a gpg error has occurred
         echo "gpg error. Do you have the Dash Core binary release signing key installed?"
      fi

      echo "gpg output:"
      # shellcheck disable=SC2001
      echo "$GPGOUT"|sed 's/^/\t/g'
      clean_up $SIGNATUREFILENAME $SIGNATUREFILENAME.2 $TMPFILE
      exit "$RET"
   fi
```

Reply:
I figured it'd be better to have "proper" syntax. The rest of the file is weirdly like 3 space tabs?

Reply:
>The rest of the file is weirdly like 3 space tabs?

yeah, not sure why. let's either keep it 3 for consistency or convert the whole thing to 2.